examples-gallery/plot_drone.py

```diff
@@ -230,7 +230,7 @@
 
 # %%
 # Plot the constraint violations.
-fig, axes = plt.subplots(2, figsize=(12.8, 9.6),
```

Comment:
These were ratio 3:4 aspect, I suspect on purpose. Is there a reason to change that?

Reply:
> These were ratio 3:4 aspect, I suspect on purpose. Is there a reason to change that?

No, I did not give any thought. I just changed to make it run with my proposal